mautic/mautic
# Allow translation of language name into other languages

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #5110 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Add some translations of language names - French for French is "Francais", for German is "Allemande" etc.
2. Login and got to the account settings page.
3. Change you language to French, apply and refresh.
4. The selected language should be "Francais"
5. Look in the language dropdown, see the "Allemande" (German) is there, choose it.
6. Apply and refresh.
7. The selected language says "Deutsche".
8. Look in the dropdown for "Türkisch" - it should be there.

#### List backwards compatibility breaks:
1. Perhaps this should be a system option - because some installs currently with the system language set to other than English, specially those that use another script (Hindi, East Asian languages...) may or may not want the dropdown list to be localized. e.g. for a non-Chinese person to choose their language from a dropdown list that is all in Chinese script is "difficult". And the same the other way, a KOrean person who does not know latin script struggles to select a language.
Note: "English - United States" seems to "stick" in the list. So that is some sort of common reference point. If I arrive on a Chinese installation I can first choose "English - United States" then pick the language I really want from the dropdown list in English.
